### PR TITLE
Throw an error when a chatnet has no available url

### PR DESCRIPTION
**Pull request by LemonBoy**
I spent too much time thinking `IRCNet` servers were shit but it turns out I had no server block for it and irssi kept trying to connect to my ISP page-not-found redirect.

Hilarious :rocket:

### CODE REVIEW

File: src/core/chat-commands.c [hunk 1]
```diff
@@ -73,6 +73,12 @@ static SERVER_CONNECT_REC *get_server_connect(const char *data, int *plus_addr,
 
 	conn = server_create_conn(proto != NULL ? proto->id : -1, addr,
 				  atoi(portstr), chatnet, password, nick);
+	if (conn == NULL) {
+		g_warning("Cuntnugget");
```
Comment (dequis) [inline src/core/chat-commands.c]: uhm...

Comment (LemonBoy) [inline src/core/chat-commands.c]: Well, if I get a ACK on the behaviour I'll add a new CMDERR_ later

**Comment by ailin-nemui** [comment]
We need to raise abi version due to formats addition

**Comment by ailin-nemui** [comment]
lgtm

File: src/core/servers-setup.c [hunk 2]
```diff
@@ -323,6 +323,8 @@ server_create_conn(int chat_type, const char *dest, int port,
 		rec = create_chatnet_conn(chatrec->name, port, password, nick);
```
Comment (ailin-nemui) [inline src/core/servers-setup.c]: just return rec; ?

**Comment by LemonBoy** [comment]
PR updated and ready to :rocket:

**Pull request merged**